src/eynollah/eynollah.py

```diff
@@ -5276,104 +5281,101 @@ def run(self):
-                                            cropped_lines_meging_indexing.append(-1)
-                                        else:
+                                    if not self.export_textline_images_and_text:
+                                        if h2w_ratio > 0.05:
                                             img_fin = self.preprocess_and_resize_image_for_ocrcnn_model(img_crop, image_height, image_width)
                                             cropped_lines.append(img_fin)
                                             cropped_lines_meging_indexing.append(0)
-                                    #img_crop = tf.reverse(img_crop,axis=[-1])
-                                    #img_crop = self.distortion_free_resize(img_crop, img_size)
-                                    #img_crop = tf.cast(img_crop, tf.float32) / 255.0
-                                    
-                                    #cropped_lines.append(img_crop)
-
-                    indexer_text_region = indexer_text_region +1
+                                        else:
+                                            splited_images = self.return_textlines_split_if_needed(img_crop)
+                                            if splited_images:
+                                                img_fin = self.preprocess_and_resize_image_for_ocrcnn_model(splited_images[0], image_height, image_width)
+                                                cropped_lines.append(img_fin)
+                                                cropped_lines_meging_indexing.append(1)
+                                                img_fin = self.preprocess_and_resize_image_for_ocrcnn_model(splited_images[1], image_height, image_width)
+                                                
+                                                cropped_lines.append(img_fin)
+                                                cropped_lines_meging_indexing.append(-1)
+                                            else:
+                                                img_fin = self.preprocess_and_resize_image_for_ocrcnn_model(img_crop, image_height, image_width)
+                                                cropped_lines.append(img_fin)
+                                                cropped_lines_meging_indexing.append(0)
+                                        
+                                if self.export_textline_images_and_text:
+                                    if child_textlines.tag.endswith("TextEquiv"):
+                                        for cheild_text in child_textlines:
+                                            if cheild_text.tag.endswith("Unicode"):
+                                                textline_text = cheild_text.text
+                                                if not textline_text:
+                                                    pass
+                                                else:
```

Comment:
Why not

```python
if textline_text:
```

Reply:
I have no idea why I did that :D I’ll make the changes in the next commit.